rpc_jobs/standard_job_premerge.yml

```diff
@@ -84,80 +84,100 @@
       env.RE_JOB_REPO_NAME = "{repo_name}"
 
       String credentials = "{credentials}"
+      String skip_pattern = "{skip_pattern}"
 
-      common.standard_job_slave(env.SLAVE_TYPE) {{
-        // Set the default environment variables used
-        // by the artifact and test result collection.
-        env.RE_HOOK_ARTIFACT_DIR="${{env.WORKSPACE}}/artifacts"
-        env.RE_HOOK_RESULT_DIR="${{env.WORKSPACE}}/results"
+      Boolean run_test
```

Comment:
I think this defaults `run_test` to null, which is false.  Would it be safer to explicitly default it to true?

Reply:
That's a fair point, the stage "should" always run but Jenkins is not one to be trusted. I've made the following change
```diff
diff --git a/rpc_jobs/standard_job_premerge.yml b/rpc_jobs/standard_job_premerge.yml
index da02428..55b56d8 100644
--- a/rpc_jobs/standard_job_premerge.yml
+++ b/rpc_jobs/standard_job_premerge.yml
@@ -86,7 +86,7 @@
       String credentials = "{credentials}"
       String skip_pattern = "{skip_pattern}"
 
-      Boolean run_test
+      Boolean run_test = true
       // We need to checkout the repo on the CIT Slave so that
       // we can check whether the patch only includes changes
       // in files matching the skip pattern. If that is the case,
```